src/ExtensionsSample/Samples/EasyTableSamples.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.EasyTables;
+using Microsoft.WindowsAzure.MobileServices;
+
+namespace ExtensionsSample
+{
+    public static class EasyTableSamples
+    {
+        // POCO output binding 
+        //   The binding inserts the newly created item into the Easy Table when the 
+        //   method successfully exits.         
+        public static void InsertItem([EasyTable] out Item newItem)
+        {
+            newItem = new Item()
+            {
+                Id = Guid.NewGuid().ToString(),
+                Text = new Random().Next().ToString()
+            };
+        }
+
+        // Query input binding 
+        //   The binding creates a strongly-typed query against the Item table. 
+        //   The binding does not do anything with the results when the function exits.  
+        //  This example takes the results of the query and puts them in a queue for further 
+        //  processing. 
+        public static async Task EnqueueItemToProcess(
```

Comment:
I'm not seeing any call to UseEasyTables anymore in this review - I assume that is intentional? Probably for the best I think, since it would then require users of this sample to have went through all the provisioning steps to go create a MobileService and all that.